### PR TITLE
Use semantic_version to compare versions in /v1/query/get_latest

### LINKED ISSUE #44: wrap thinks zlib 1.2.8 is the latest, 1.2.11 is
Which I assume means that somewhere sort algorithm that thinks that `2 >19` is in use.

### PR DESCRIPTION
**Pull request by sarum9in**
Closes #44.

### CODE REVIEW

File: mesonwrap/wrapdb.py [hunk 1]
```diff
@@ -49,13 +50,15 @@ def name_search(self, text):
 
     def get_versions(self, project, latest=False):
         c = self.conn.cursor()
-        qtempl = '''SELECT branch, revision FROM wraps WHERE project == ? ORDER BY branch DESC, revision DESC %s;'''
+        query = '''SELECT branch, revision FROM wraps WHERE project == ? ORDER BY branch DESC, revision DESC;'''
+        c.execute(query, (project,))
         if latest:
-            query = qtempl % 'LIMIT 1'
+            # TODO consider computing this during import
+            l = max(map(lambda r: (semantic_version.Version(r[0]), r[1]),
```
Comment (ignatenkobrain) [inline mesonwrap/wrapdb.py]: __[Flake8]__

[E741] ambiguous variable name 'l'

**Comment by sarum9in** [comment]
Isn't it deprecated? https://docs.python.org/3/library/distutils.html
https://docs.python.org/3/distributing/index.html#key-terms
"direct use of distutils is being phased out"

**Comment by nirbheek** [comment]
fwict it's replaced by setuptools but it's not going away.

**Comment by sarum9in** [comment]
@nirbheek alright, I updated the code. Also since we need to support non-conforming projects we have to use `LooseVersion`, see tests.

**Pull request merged**